utils.groovy

```diff
@@ -30,12 +30,23 @@ def load_pipecfg() {
     return readYaml(file: "pipecfg/config.yaml")
 }
 
+def substitute_str(str, stream, buildid = "") {
+    def date = shwrapCapture("date +%Y-%m-%d")
+    def basearch = shwrapCapture("cosa basearch")
```

Comment:
This probably needs to be a parameter too because i.e. we'd want to be able to operate against `aarch64` even though we're on `x86_64` jenkins.